buildtest/cli/compilers.py

```diff
@@ -56,9 +56,12 @@ def compiler_test(configuration):
     fail_compilers = []
 
     compilers = configuration.target_config["compilers"]["compiler"]
+    target_compilers = set(specific_compilers)
 
     for name in compilers:
         for module in compilers[name]:
+            if target_compilers and module not in target_compilers:
+                continue
```

Comment:
it could be helpful to have a `console.print(f"Skipping test for compiler {target_compilers}")`  message